CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+## 2023-01-16 v0.14.0
+
+Release highlights:
+- An important fix for the Waku message archive returning inconsistent responses to history queries.
+- Support for [AutoNAT](https://docs.libp2p.io/concepts/nat/autonat/) and [libp2p Circuit Relay](https://docs.libp2p.io/concepts/nat/circuit-relay/) that allows, among other things, for [NAT hole punching](https://docs.libp2p.io/concepts/nat/hole-punching/).
+- Support for structured logging in JSON format.
+- A fix for an underlying file descriptor leak that affected websocket connections.
+
+### Features
+
+- Support for [AutoNAT](https://docs.libp2p.io/concepts/nat/autonat/)
```

Comment:
just a preference, but like having a link to the PR to see the changes (like in the fixes section)

- Support for [AutoNAT](https://docs.libp2p.io/concepts/nat/autonat/) [#1472](https://github.com/waku-org/nwaku/pull/1472)

Not sure if you could try the [autogenerated](https://github.com/waku-org/nwaku/pull/1378#discussion_r1021893395) notes. With good PR namings it makes creating the release notes quite easy :D

Reply:
Thanks! I do use the autogenerated notes as starting point, but find that to make it humanly interpretable requires significant rewrite (e.g. many "features" and "fixes" just fall under a "updated submodules"). It was a bit easier to rewrite and keep links to PRs related to fixes. I think this is the way to go in the future when we have automated releases, but we may need to revisit how commit messages are formatted.

Reply:
sure, not suggesting to use the autogenerated notes by default. +1 for human readable.
my only concern was to link the PR of each feature :)